policyengine_canada/tests/gov/cra/benefits/dental_benefit/dental_benefit_eligible.yaml

```diff
@@ -0,0 +1,45 @@
+- name: Child under 12 without insurance and with dental expenses; eligible
+  period: 2022
+  input:
+    people:
```

Comment:
Single this is a person-level field, please make all these tests person-level only (remove the `people` and `households` bit and just provide the three input variables). Also make one of the tests age 11.